src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CancelVideoBookingServiceTest.kt

```diff
@@ -31,6 +34,7 @@ class CancelVideoBookingServiceTest {
     booking.statusCode isEqualTo StatusCode.CANCELLED
 
     verify(videoBookingRepository).saveAndFlush(booking)
+    verify(bookingHistoryService).createBookingHistory(any(), any())
```

Comment:
Why not `verify(bookingHistoryService).createBookingHistory(HistoryType.CANCEL, booking) `?

Reply:
For some reason the booking is not the object it expects... id is different. Assume something is creating a new version of it.

Reply:
That was the case on Amend anyway, so might be an assumption here - I'll check.

Reply:
Not true here - so added the specific expected items and removed the any().